### DISCUSSION TITLE
Deploy de API com FRONT

### DISCUSSION THREAD

**Discussion by arturn1** [balta-io/7196#43] [Perguntas e Respostas] [answered]
Finalizei o Projeto, porem não estou conseguindo fazer o deploy na Azure.
Criei o do Front End via StandAlone React App, e associei ao projeto, mas nao consigo acessar, apenas o Swagger, fica disponivel.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @arturn1, como vai?

Infelizmente só com estas informações não consigo te ajudar. Preciso de mais detalhes para saber o que pode ser.
